src/main/pages/che-7/end-user-guide/proc_configuring-global-gradle-repositories.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// using-gradle-artifact-repositories
+
+[id="configuring-global-gradle-repositories_{context}"]
+= Configuring global Gradle repositories
+
+You can use an initialization script to configure global repositories for the workspace.
```

Comment:
Is the init script just one of the options for configuring the repos? If yes, what are the other ones? If not, I suggest using only:

"Use an initialization script to..."